bin/resources/GameIndex.yaml

```diff
@@ -1765,6 +1765,9 @@ SCED-50381:
 SCED-50404:
   name: "Official PlayStation 2 Magazine Demo 10" # German
   region: "PAL-E-G"
+SCED-50450:
+  name: "Le Monde des Bleus 2002"
```

Comment:
shouldn't this have [Demo] on the end?

Reply:
![image](https://user-images.githubusercontent.com/24227051/214359480-e4890746-afca-4802-8140-077bceb73a4e.png)

Seems like it.

Reply:
_You see noooothing_